### PR TITLE
add devise-bootstrap-views gem and generate view

### PR DESCRIPTION
**Pull request by LeeRenJie**
use `rails generate devise:views:bootstrap_templates` to generate views for forms

### CODE REVIEW

File: finance-tracker/app/views/devise/shared/_links.html.erb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
+  <%- if controller_name != 'sessions' %>
+    <%= link_to t(".sign_in"), new_session_path(resource_name) %><br />
+  <% end -%>
```
Comment (tlkiong) [inline finance-tracker/app/views/devise/shared/_links.html.erb:4]: Why use `-%>` syntax?

**Review by tlkiong — CHANGES_REQUESTED** [review]
Why use this gem and not build it yourself?

**Comment by LeeRenJie** [comment]
> Why use this gem and not build it yourself?

The course wanna show us that if we use devise and hopes to use bootstrap, there is an alternative to that to style the forms. 😅

File: finance-tracker/app/views/devise/shared/_links.html.erb (hunk 1 above)
Comment (LeeRenJie) [reply finance-tracker/app/views/devise/shared/_links.html.erb:4]: From what I research, it is to remove the next character if it is a newline `\n`, but then after deleting this, I see no difference in views. Are there any other reasons why they use this syntax?

Comment (tlkiong) [reply finance-tracker/app/views/devise/shared/_links.html.erb:4]: to remove the `\n`

Comment (LeeRenJie) [reply finance-tracker/app/views/devise/shared/_links.html.erb:4]: but then I dont see any `\n` in the code, is it okay to remove this?

**Review by tlkiong — APPROVED** [review]

**Pull request merged**